components/event-bus/cmd/event-bus-publish-knative/dockerBuild.sh

```diff
@@ -9,9 +9,12 @@ COMPONENT=$3
 echo -e "Start building docker image...."
 mkdir -p docker/image
 cp Dockerfile docker/image/
-cp -R main.go application httpserver handlers docker/image/
-cp -R ../../internal/publish-knative docker/image/
+cp -R main.go application httpserver handlers publisher validators docker/image/
+cp -R ../../internal/knative/publish docker/image/
```

Comment:
Do we need this step?

Reply:
Yes, otherwise `make docker-build` fails to execute.

Reply:
Done